freqtrade/freqai/freqai_interface.py

```diff
@@ -563,8 +563,11 @@ def model_exists(self, dk: FreqaiDataKitchen) -> bool:
             file_type = ".joblib"
         elif self.dd.model_type == 'keras':
             file_type = ".h5"
-        elif 'stable_baselines' in self.dd.model_type or 'sb3_contrib' == self.dd.model_type:
+        elif ('stable_baselines' in self.dd.model_type or
+              'sb3_contrib' == self.dd.model_type or
+              'pytorch' == self.dd.model_type):
```

Comment:
```suggestion
        elif (self.dd.model_type in ["stable_baselines", "sb3_contrib", "pytorch"]):
```

Reply:
this statements are not equivalent.
'stable_baselines' in self.model_type vs self.model_type in "stable_baselines"

Reply:
Please keep the suggested change, but you are correct that we simply need to add a "3" to the end of `stable_baselines3`. Please follow same convention as suggested in the DataDrawer.